lib/cib/cib_attrs.c

```diff
@@ -591,7 +591,7 @@ set_standby(cib_t * the_cib, const char *uuid, const char *scope, const char *st
 
     } else {
         scope = XML_CIB_TAG_NODES;
-        attr_id = crm_strdup_printf("standby-%.256s", uuid);
+        attr_id = crm_strdup_printf("nodes-%.256s-standby", uuid);
```

Comment:
The reason is that current `nodes-1-standby` conflicts with `standby-1`, so `crm` cannot recognize which attribute to change

Consider we have 
```
<nvpair id="nodes-1-standby" name="standby" value="off"/>
<nvpair id="standby-1" name="standby" value="on"/>
```
in cib.

for example, trying to set node online via `crm` causes an error

```
$ crm node online
Multiple attributes match name=standby
  Value: off 	(id=nodes-1-standby)
  Value: on 	(id=standby-1)
Error performing operation: Name not unique on network
```

Btw, there is also no way to change attribute via `crm_attribute` by id.